cpp/qrcodegen.cpp

```diff
@@ -355,7 +355,9 @@ QrCode::QrCode(int ver, Ecc ecl, const vector<uint8_t> &dataCodewords, int msk)
 	drawFormatBits(msk);  // Overwrite old format bits
 	
 	isFunction.clear();
-	isFunction.shrink_to_fit();
+	//isFunction.shrink_to_fit();
+
```

Comment:
what is the purpose of this commented functions? Can they be removed?

Reply:
shrink_to_fit() is for C++ 11, to remove the extra capacity inside a vector. 
Replaced by swap() for C++98, same purpose.